common/common.go

```diff
@@ -226,3 +250,24 @@ func readResponses(responses chan transmission.Response) {
 		}
 	}
 }
+
+func fieldMappingsFrom(environmentName string) (map[string]string, error) {
```

Comment:
This style of function naming isn't too common in go, I'd go with something like `getFieldMappings`.